### PR TITLE
setting multiline directives, flagging markdown and updating labels f…

### PR DESCRIPTION
**Pull request by olinux**
…or abbreviations

### CODE REVIEW

File: vocab/properties.json [hunk 1]
```diff
@@ -5657,7 +5657,7 @@
       "inVersions": [
         "latest"
       ],
-      "multiline": false
+      "multiline": true
```
Comment (lzehl) [inline vocab/properties.json:5660]: this one should be markdown as well

File: vocab/properties.json [hunk 2]
```diff
@@ -7434,7 +7434,7 @@
...
```
Comment (lzehl) [inline vocab/properties.json:7437]: this one should be also markdown

**Comment by olinux** [comment]
@lzehl : requested changes are applied. Are you also ok with the color definitions (as discussed, they can still be streamlined in a later moment)?

**Review by lzehl — APPROVED** [review]
@olinux looks good :) I'm fine with the colors as they are right now. We can adapt then later when we want to

**Pull request merged**